rust/codecs/src/cli.rs

```diff
@@ -38,12 +39,30 @@ struct List;
 impl List {
     async fn run(self) -> Result<()> {
         let mut table = table::new();
-        table.set_header(["Name", "Status"]);
+        table.set_header(["Name", "Input formats", "Output formats", "Status"]);
 
         for codec in super::list() {
+            let supports_input = codec.supports_from_path();
+            let supports_output = codec.supports_to_path();
+            let status = codec.status();
             table.add_row([
                 Cell::new(codec.name()).add_attribute(Attribute::Bold),
-                Cell::new(codec.status()),
+                match supports_input {
+                    true => Cell::new("Yes").fg(Color::Green),
+                    false => Cell::new("No"),
+                },
+                match supports_output {
+                    true => Cell::new("Yes").fg(Color::Green),
+                    false => Cell::new("No"),
+                },
```

Comment:
Rather than `Yes/No`, I think we need a column of the formats that the codec supports and a single column indicating if it supports `to/from` those formats (I suggest those words to link to the words we are using for the `--to` and `--from` options elsewhere).

Reply:
I realized that we already had `supports_from_formats` and `supports_to_formats` so went ahead and implemented this already.